src/preamble_minimal.js

```diff
@@ -61,14 +62,15 @@ Module['wasm'] = base64Decode('<<< WASM_BINARY_DATA >>>');
 #include "runtime_functions.js"
 #include "runtime_strings.js"
 
-var HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64;
+var _HEAP_DATA_VIEW, HEAP8, HEAP16, HEAP32, HEAPU8, HEAPU16, HEAPU32, HEAPF32, HEAPF64;
 var wasmMemory, buffer, wasmTable;
 
 function updateGlobalBufferAndViews(b) {
 #if ASSERTIONS && USE_PTHREADS
   assert(b instanceof SharedArrayBuffer, 'requested a shared WebAssembly.Memory but the returned buffer is not a SharedArrayBuffer, indicating that while the browser has SharedArrayBuffer it does not have WebAssembly threads support - you may need to set a flag');
 #endif
   buffer = b;
+  _HEAP_DATA_VIEW = new DataView(b);
```

Comment:
All the usages of this new variable should be `#ifdef`ed so that they do not appear in a non-big-endian build.

Reply:
Done